README.md

```diff
@@ -1,6 +1,6 @@
 # Members Introduction
 
 ## Neque Porro
-- DoB: February 24, 1955
-- Email: steve@email.com
-- Hobbies: Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...
+- DoB: September 23, 1994.
+- Email: ryo.asaka@g.pascalia.co.jp
+- Hobbies: Sports
```

Comment:
Let keep current information and add your profile at bottom of README.md instead of replace old content, please fix it. For example, the file should be like:

```
## Neque Porro
- DoB: February 24, 1955
- Email: steve@email.com
- Hobbies: Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...
# Ryo Asaka
- DoB: September 23, 1994.
- Email: ryo.asaka@g.pascalia.co.jp
- Hobbies: Sports